app/views/users/_form.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= form_with model: @user do |form| %>
+<% @user.errors.full_messages_for(:username).each do |message| %>
+<div class="alert alert-danger alert-dismissible fade show">
+    <%= message %>
+</div>
+<% end %>
+<div class="form-group">
+    <%= form.label :username, 'Write Username:' %>
+    <%= form.text_field :username, placeholder: 'eg. John Doe',class: 'form-control' %>
+</div>
+<% @user.errors.full_messages_for(:email).each do |message| %>
+<div class="alert alert-danger alert-dismissible fade show">
+    <%= message %>
+</div>
+<% end %>
+<div class="form-group">
+    <%= form.label :email, 'Write email:' %>
+    <%= form.email_field :email, placeholder: 'eg. John@doe.com',class: 'form-control' %>
+</div>
+<% @user.errors.full_messages_for(:password).each do |message| %>
+<div class="alert alert-danger alert-dismissible fade show">
+    <%= message %>
+</div>
+<% end %>
+<% if @mode != "edit"%>
+<div class="form-group">
+    <%= form.label :password, 'Write password:' %>
+    <%= form.password_field :password, placeholder: 'eg. 123456',class: 'form-control' %>
+</div>
+<% end %>
+<div class="text-center">
+    <%= link_to 'back', root_path, class: 'btn btn-secondary'%>
+    <%= form.submit class: 'btn btn-primary'%>
+</div>
```

Comment:
Great work using partial for your forms and linking it to the edit and the new pages. 👏🏿 🥳 

However, you have left out a hard requirement for this project. Kindly check below ⬇️ 

- [ ] Kindly create an HTML form and comment it out. This is a hard requirement for this project. It will help you visualize the similarities and differences between the two forms. ⏬ 

![user_form1](https://user-images.githubusercontent.com/32361963/113702558-58b58b00-96e2-11eb-92b0-ab8c86422466.png)
![user_form2](https://user-images.githubusercontent.com/32361963/113702629-6cf98800-96e2-11eb-8e76-cd10b9910e61.png)